### PR TITLE
[AERIE-1874] Add command expansion test into e2e tests

### PR DESCRIPTION
**Pull request by dyst5422**
* **Tickets addressed:** AERIE-1874
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

Removed auth for command expansion tests and added an e2etest task for that project.

## Verification

Ran e3etests locally

## Documentation

None

### CODE REVIEW

**Comment by dyst5422** [comment]
Looks like we're missing permissions on a directory creation?
https://github.com/NASA-AMMOS/aerie/runs/6927902592?check_suite_focus=true#step:13:23

```
EACCES: permission denied, mkdir '/usr/src/app/commanding_file_store/aerie/'
```

I'm definitely out of my depth with this one, does one of our CI gods want to chime in/point me in the right direction?

**Comment by dyst5422** [comment]
Apparently, it was just an issue of trying to mount the local code directory as we do for our development docker-compose. Everything looks good now though!

**Review by mattdailis — APPROVED** [review]

File: command-expansion-server/jestGlobalSetup.js [hunk 1]
```diff
@@ -11,57 +11,59 @@ process.env.MERLIN_GATEWAY_URL = 'http://localhost:9000';
...
-        if (!this.muted) process.stdout.write(chunk, encoding);
-        callback();
-      },
-    });
-    const readlineInterface = readline.createInterface({
-      input: process.stdin,
-      output: mutableStdout,
-      terminal: true,
-    });
+  // Removing this because it appears we don't need it with auth: none, keeping it here in case we need it in the future though
```
Comment (patkenneally) [inline command-expansion-server/jestGlobalSetup.js]: Just a thought: My preference would be to use the Git history to retrieve this if it is indeed needed in the future. My thinking here is that as we get further away from the time when we first commented out code, we slowly lose the context of why  we did so. The Git history with commit messages is a good way to document those decisions. I worry about a code base becoming sprinkled with commented code over time.

**Review by patkenneally — APPROVED** [review]

File: command-expansion-server/jestGlobalSetup.js (hunk 1 above)
Comment (dyst5422) [reply command-expansion-server/jestGlobalSetup.js]: Fair point. I guess the risk there is that the existence of this code is forgotten when hidden in the git history. Not sure which point should win out in this case.

Comment (dyst5422) [reply command-expansion-server/jestGlobalSetup.js]: I removed it - authenticating for tests isn't something I think we generally want, so I can't imagine this code being worth keeping around

Comment (patkenneally) [reply command-expansion-server/jestGlobalSetup.js]: Myself not being very familiar with this particular test set up, is there a way to factor out the login code to say a separate file that can be kept along side and selectively used?

Comment (patkenneally) [reply command-expansion-server/jestGlobalSetup.js]: > I removed it 

Okay 👍

**Pull request merged**